dymaptic.Pro.ZoomToCoordinates/Views/About.xaml

```diff
@@ -0,0 +1,51 @@
+﻿<controls:ProWindow x:Class="dymaptic.Pro.ZoomToCoordinates.Views.About"
+        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+        xmlns:controls="clr-namespace:ArcGIS.Desktop.Framework.Controls;assembly=ArcGIS.Desktop.Framework"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+        xmlns:extensions="clr-namespace:ArcGIS.Desktop.Extensions;assembly=ArcGIS.Desktop.Extensions"
+        mc:Ignorable="d"
+        Title="About" Height="380" Width="350"
+        WindowStartupLocation="CenterOwner"
+    >
+    <controls:ProWindow.Resources>
+        <ResourceDictionary>
+            <ResourceDictionary.MergedDictionaries>
+                <extensions:DesignOnlyResourceDictionary Source="pack://application:,,,/ArcGIS.Desktop.Framework;component\Themes\Default.xaml"/>
+            </ResourceDictionary.MergedDictionaries>
+        </ResourceDictionary>
+    </controls:ProWindow.Resources>
+    <Grid>
+        <Grid.RowDefinitions>
+            <RowDefinition Height="auto"/>
+            <RowDefinition Height="auto"/>
+            <RowDefinition Height="auto"/>
+        </Grid.RowDefinitions>
+
+        <!-- Application Name and Version -->
+        <StackPanel Grid.Row="0" HorizontalAlignment="Center" VerticalAlignment="Top">
+            <TextBlock Text="Zoom to Coordinates" FontSize="24" FontWeight="Bold" />
+            <TextBlock Text="Version 1.1" FontSize="16" FontStyle="Italic" />
+        </StackPanel>
+
+        <!-- Application and Company Description -->
+        <StackPanel Grid.Row="1" Margin="15">
+            <TextBlock TextWrapping="Wrap" >
+                The Zoom to Coordinates Add-In, created by Dymaptic, is an open-source solution for ArcGIS Pro, fully compatible with version 3.x. It is 
+                released under the permissive MIT license, and its source code can be accessed on
+                <Hyperlink NavigateUri="https://github.com/dymaptic/dy-pro-zoom-to-coordinates" RequestNavigate="Hyperlink_RequestNavigate">GitHub</Hyperlink>.
+                <LineBreak/>
+                <LineBreak/>
+                Dymaptic is a leading woman-owned GIS software development company and an Esri Silver Partner that specializes in providing innovative and 
+                tailored GIS solutions including custom Add-In development. To learn more about dymaptic and explore our cutting-edge solutions, visit our 
+                homepage at
+                <Hyperlink  NavigateUri="https://www.dymaptic.com/" RequestNavigate="Hyperlink_RequestNavigate">dymaptic.com</Hyperlink>.
+            </TextBlock>
+        </StackPanel>
+
+        <!-- Dymaptic Company Logo -->
+        <Image Grid.Row="2" HorizontalAlignment="Center" Height="75" Width="300"
```

Comment:
You should save images close to the useful dimensions (here `75x300`), the giant image size is just wasted storage/package size otherwise.